apps/web/components/sections/project/project-showcase.tsx

```
@@ -0,0 +1,52 @@
+            className="absolute top-4 right-4 text-white text-2xl font-bold"
+          >
+            &times;
+          </button>
+          {selectedMedia.type === "image" ? (
+            <img src={selectedMedia.src} alt="Showcase" className="max-w-full max-h-full rounded-lg" />
+          ) : (
+            <video
+              src={selectedMedia.src}
+              controls
+              className="max-w-full max-h-full rounded-lg"
+            />
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)